Resources/Prototypes/_NF/Catalog/pirate_uplink_catalog.yml

```diff
@@ -0,0 +1,302 @@
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateHoverbike
+  name:  uplink-pirate-hoverbike-name
+  description: uplink-pirate-hoverbike-desc
+  productEntity: VehicleHoverbikePirateKey
+  icon: { sprite: _NF/Objects/Vehicles/hoverbike.rsi, state: keys }
+  cost:
+    Doubloon: 10
+  categories:
+    - UplinkPirateUtility
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+    - !type:BuyerJobCondition
+      whitelist:
+        - PirateCaptain
+
+- type: listing
+  id: UplinkPirateBlunderbuss
+  name:  uplink-pirate-blunderbuss-name
+  description: uplink-pirate-blunderbuss-desc
+  productEntity: WeaponShotgunBlunderbuss
+  icon: { sprite: Objects/Weapons/Guns/Shotguns/blunderbuss.rsi, state: icon }
+  cost:
+    Doubloon: 2
+  categories:
+    - UplinkPirateWeapons
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateRevolver
+  name:  uplink-pirate-revolver-name
+  description: uplink-pirate-revolver-desc
+  productEntity: WeaponRevolverPirate
+  icon: { sprite: Objects/Weapons/Guns/Revolvers/pirate_revolver.rsi, state: icon }
+  cost:
+    Doubloon: 1
+  categories:
+    - UplinkPirateWeapons
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateCannon
+  name:  uplink-pirate-cannon-name
+  description: uplink-pirate-cannon-desc
+  productEntity: WeaponLauncherPirateCannon
+  icon: { sprite: Objects/Weapons/Guns/Launchers/pirate_cannon.rsi, state: icon }
+  cost:
+    Doubloon: 6
+  categories:
+    - UplinkPirateWeapons
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkFlintlock
+  name:  uplink-pirate-flintlock-name
+  description: uplink-pirate-flintlock-desc
+  productEntity: WeaponPistolFlintlock
+  icon: { sprite: Objects/Weapons/Guns/Snipers/flintlock.rsi, state: base }
+  cost:
+    Doubloon: 2
+  categories:
+    - UplinkPirateWeapons
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+    - !type:BuyerJobCondition
+      whitelist:
+        - PirateCaptain
+
+- type: listing
+  id: UplinkEnergyCutlass
+  name:  uplink-pirate-ecutlass-name
+  description: uplink-pirate-ecutlass-desc
+  productEntity: EnergyCutlass
+  icon: { sprite: Objects/Weapons/Melee/e_cutlass.rsi, state: e_cutlass }
+  cost:
+    Doubloon: 2
+  categories:
+    - UplinkPirateWeapons
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+    - !type:BuyerJobCondition
+      whitelist:
+        - PirateCaptain
+
+- type: listing
+  id: UplinkPirateShotgunAmmo
+  name:  uplink-pirate-shotgunammo-name
+  description: uplink-pirate-shotgunammo-desc
+  productEntity: BoxMagazineShotgun
```

Comment:
Is this supposed to be the shotgun slug dispenser?  This is not the shotgun slug dispenser.  I can change it on my end, but seems odd to get a bunch of bulldog mags.